src/TConsole.cpp

```diff
@@ -613,7 +617,7 @@ void TConsole::resizeEvent(QResizeEvent* event)
     int y = event->size().height();
 
 
-    if (!mIsSubConsole && !mIsDebugConsole) {
+    if (mType & (MainConsole|Buffer)) {
```

Comment:
I think this and the previous can be run together with the two `event->size()` lines being done beforehand.